manifests/t/Tonec/InternetDownloadManager/6.41.7.1/Tonec.InternetDownloadManager.locale.en-US.yaml

```
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Tonec.InternetDownloadManager
+PackageVersion: 6.41.7.1
```

```suggestion
PackageVersion: 6.41.7
```
Must match Control Panel / Apps&Features